packages/stack/src/views/Stack/CardContainer.tsx

```diff
@@ -128,6 +130,38 @@ function CardContainer({
 
   const { colors } = useTheme();
 
+  const [pointerEvents, setPointerEvents] = React.useState<'box-none' | 'none'>(
+    'box-none'
+  );
+
+  const valueListenerCallback = React.useCallback(
+    ({ value }: { value: number }) => {
+      const newIsUnderEpsilon = value <= EPSILON ? 'box-none' : 'none';
+      if (newIsUnderEpsilon !== pointerEvents) {
```

Comment:
You don't need to check, afaik React will skip the update if the value is the same (only for `useState` hook). BTW it's better to use the updater callback to check previous state.